common/presenters/populationTable/location-meta-to-spaces-link.js

```diff
@@ -0,0 +1,19 @@
+const spacesFormatter = function (count) {
+  if (count === undefined) {
```

Comment:
Disclaimer: Personal preference

But you could use a switch statement here too. I personally find them easier to read than lots of `else if`.

Reply:
This could also be switched out to use i18n and the plural feature mentioned below:

```javascript
const i18n = require('../config/i18n')
const spaces = i18n.t('spaces', { count })

return count === undefined ? i18n.t('add_space') : spaces
```

```json
{
  ...
  "spaces": "{{count}} space",
  "spaces_plural": "{{count}} spaces",
  ...
}
```

Reply:
Replaced this block with that `i18n` plural feature